HW_1/task_3.py

```diff
@@ -0,0 +1,16 @@
+# Задача №3 из домашнего задания №1.
+
+# Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. 
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. 
+# Вам требуется написать программу, которая проверяет счастливость билета.
+# Пример:
+# 385916 -> yes
+# 123456 -> no
+
+ticket = list(map(int, list(input())))
```

Comment:
Переводить в список введенную строку не обязательно.
```suggestion
ticket = list(map(int, input()))
```